AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/MySqlUserRepository.cs

```diff
@@ -18,44 +20,68 @@ public class MySqlUserRepository : IUserRepository
         _userConverter = userConverter;
     }
 
-    public async Task<Model.User> GetUser(string userId)
+    public async Task<bool> AuthenticateUser(string userId, string password)
     {
-        return _userConverter.Convert(
-            await _settingsContext.Users.SingleAsync(u => u.UserId == userId)
-        );
+        try
+        {
+            var user = await _settingsContext.Users.SingleAsync(u =>
+                u.Id == userId &&
+                u.Password == password
+            );
+
+            return true;
+        }
+        catch (Exception ex)
```

Comment:
Could probably do a custom exception for this but shouldn't really matter for our end product